ceryx/Dockerfile

```diff
@@ -15,12 +18,17 @@ RUN curl -fSslL https://github.com/jwilder/dockerize/releases/download/${DOCKERI
 # Install lua-resty-auto-ssl for dynamically generating certificates from LE
 # https://github.com/GUI/lua-resty-auto-ssl
 RUN /usr/local/openresty/luajit/bin/luarocks install lua-resty-auto-ssl 0.12.0
+RUN mkdir /etc/resty-auto-ssl/
+RUN chown -R $user:$group /etc/resty-auto-ssl/
 
-# Add the entrypoint script
-COPY ./bin/entrypoint.sh /entrypoint.sh
-ENTRYPOINT [ "/entrypoint.sh" ]
+RUN rm -f /usr/local/openresty/nginx/conf/*
```

Comment:
We could bundle all these `RUN` commands into one, using `&&`

Reply:
I like keeping semantically different `RUN`s separate. Besides making it easier for me to understand what I am doing there, it can also provide preciser caching.

I took the opportunity though and grouped these `RUN`s: https://github.com/sourcelair/ceryx/pull/58/commits/05a27ababa79c07a62de7bd54db8c302837e66c1 and also added a comment here: https://github.com/sourcelair/ceryx/pull/58/commits/2c4bc049d01116ca4c21afd721150603e63e3946.